remote-method/index.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var typeChoices =helpers.getTypeChoices;
+var accepts = [];
+var returns = [];
+var https = [];
+
+module.exports = yeoman.generators.Base.extend({
+  // This generator does not track file changes via yeoman,
+  // as loopback-workspace is editing (modifying) files when
+  // saving project changes.
+  constructor: function() {
+    yeoman.generators.Base.apply(this, arguments);
+
+    this.argument('modelName', {
+      desc: 'Name of the model',
+      required: false,
+      type: String
+    });
```

Comment:
No problem I will add it back, yesterday I misunderstood your feedback, sorry for that. It's not a big issue :)